Multisig/UI/Settings/OwnerKeyManagement/LedgerOwnerKey/LedgerKeyFlow.swift

```diff
@@ -0,0 +1,118 @@
+//
+//  LedgerKeyFlow.swift
+//  Multisig
+//
+//  Created by Mouaz on 11/4/22.
+//  Copyright © 2022 Gnosis Ltd. All rights reserved.
+//
+
+import UIKit
+
+class LedgerKeyFlow: AddKeyFlow {
+    private var bluetoothController: BaseBluetoothController?
+
+    var flowFactory: LedgerKeyFlowFactory {
+        factory as! LedgerKeyFlowFactory
+    }
+
+    var parameters: AddLedgerKeyParameters? {
+        keyParameters as? AddLedgerKeyParameters
+    }
+
+    init(completion: @escaping (Bool) -> Void) {
+        super.init(factory: LedgerKeyFlowFactory(), completion: completion)
+    }
+
+    override func didIntro() {
+        selectDevice()
+    }
+
+    func selectDevice() {
+        let vc = flowFactory.selectDevice { [unowned self] deviceId, controller  in
+            bluetoothController = controller
+            addressPicker(deviceUUID: deviceId)
+        } onClose: {
+
```

Comment:
add comment `// not needed so that ledger connection can be re-established without closing the flow`